ggml/src/ggml-metal.m

```diff
@@ -272,6 +274,8 @@ static void ggml_backend_metal_device_rel(struct ggml_backend_metal_device_conte
     GGML_METAL_KERNEL_TYPE_SIN,
     GGML_METAL_KERNEL_TYPE_COS,
     GGML_METAL_KERNEL_TYPE_SUM_ROWS,
+    GGML_METAL_KERNEL_TYPE_AVG_POOL_2D_F32,
+    GGML_METAL_KERNEL_TYPE_MAX_POOL_2D_F32,
```

Comment:
```suggestion
    GGML_METAL_KERNEL_TYPE_POOL_2D_AVG_F32,
    GGML_METAL_KERNEL_TYPE_POOL_2D_MAX_F32,
```

Reply:
applied in 008484799146274784a5df692fd7b7508805be83